bundle/bundle.go

```diff
@@ -23,6 +23,7 @@ import (
 	"github.com/open-policy-agent/opa/ast"
 	"github.com/open-policy-agent/opa/format"
 	"github.com/open-policy-agent/opa/internal/file/archive"
+	"github.com/open-policy-agent/opa/internal/json/patch"
```

Comment:
This package has to be exported since it's exposed in the bundle.Bundle struct.

Reply:
Moved the patch object in the `bundle` package.